conf/conf.go

```diff
@@ -236,11 +238,61 @@ func getYamlFileFromFile(path string) ([]byte, error) {
 
 func getYamlFile(path string) ([]byte, error) {
 	if isExternalURL(path) {
-		return getYamlFileFromInternet(path)
+		return getYamlFileFromHTTP(path)
 	}
 	return getYamlFileFromFile(path)
 }
 
+// previousYamlFile is the content of the configuration file
+var previousYamlFile []byte
```

Comment:
all `Yaml` should be `YAML`.

```suggestion
var previousYAMLFile []byte
```